js-exercises/chunk-array-in-groups/chunkArrayInGroups.js

```diff
@@ -0,0 +1,12 @@
+function chunkArrayInGroups(array, chunkSize) {
+  const finalArr = [];
+  const maxIter = Math.ceil(array.length / chunkSize);
+  for (let i = 0; i < maxIter; i += 1) {
+    finalArr.push(array.splice(0, chunkSize));
```

Comment:
@bhupen13au `splice` method mutates the array. use the `slice` method instead.

Reply:
ok I will try with slice method